modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -10,6 +10,11 @@ Before installation, you can set the `networkConfig` configuration setting in th
 
 The most common use case for this functionality is to specify a static IP address on the bare-metal network, but you can also configure other networks such as a storage network. This functionality supports other NMState features such as VLAN, VXLAN, bridges, bonds, routes, MTU, and DNS resolver settings.
 
+[WARNING]
+====
+When specifying DNS resolver settings in the `networkConfig` configuration, do not set the `rotate` option. The `rotate` setting causes `api-int.<cluster_name>.<base_domain>` DNS lookup failures because Multus pods get stuck in the `ContainerCreating` state and the Multus container network interface (CNI) fails.
+====
```

Comment:
Maybe:

The use of DNS resovler `rotate` option interferes with the internal cluster API DNS resolution and is not supported.

Reply:
more than "not supported".  `rotate` should not be used.

> The use of the DNS resolver `rotate` option disrupts internal cluster API DNS resolution and is not allowed.

prohibited?  What's the standard phrase for "this will break the cluster, don't do it.".

Reply:
Hi @rbbratta .

Updated to state:

> Do not set the unsupported `rotate` option in the DNS resolver settings for your cluster, because the option disrupts the DNS resolution function of the internal API.